salt/apps/ocw/engines.sls

```diff
@@ -0,0 +1,72 @@
+{% set engines_basedir = salt.pillar.get('ocw:engines_basedir', '/mnt/ocwfileshare/OCWEngines') %}
+
+{% if salt['file.directory_exists'](engines_basedir) %}
+
+{% set mailto = salt.pillar.get('ocw:cron_mailto') %}
+
+manage_engines_conf:
+  file.managed:
+    - name: {{ engines_basedir }}/engines.conf
+    - template: jinja
+    - source: salt://apps/ocw/templates/engines.conf.jinja
+    - user: ocwuser
+    - group: ocwuser
+    - mode: 0640
+
+manage_generate_ocw_news_feeds_cronjob:
+  file.managed:
```

Comment:
We can use the cron state module for these. https://docs.saltstack.com/en/latest/ref/states/all/salt.states.cron.html#module-salt.states.cron

Reply:
I've changed this in 0b3b819